source/common/upstream/load_balancer_impl.cc

```diff
@@ -517,20 +515,39 @@ bool ZoneAwareLoadBalancerBase::earlyExitNonLocalityRouting() {
   return false;
 }
 
-bool LoadBalancerContextBase::validateOverrideHostStatus(Host::Health health,
-                                                         OverrideHostStatus status) {
-  switch (health) {
-  case Host::Health::Unhealthy:
-    return status & UnhealthyStatus;
-  case Host::Health::Degraded:
-    return status & DegradedStatus;
-  case Host::Health::Healthy:
-    return status & HealthyStatus;
+std::bitset<32> LoadBalancerContextBase::createOverrideHostStatus(
```

Comment:
Nit: can you add a `using` type alias for this, e.g. `using HostStatusSet = std::bitset<32>`? search-replace this across the diff.

Reply:
get it.